README.md

```diff
@@ -7,17 +7,53 @@ Advanced topic group AI: Ket-Hwa, Tingxu, Xiaoting, Zhuoyan
 
 Advanced topic group multiplayer: Jagger, Alli, Sean
 
-# game premise:
+## game premise:
 
 It is a 2D pixel-art platformer where cooperation is necessary. Players control several characters bound by a rope, to conquer the majestic and treacherous slopes of a mysterious mountain, you must master the art of collaborative climbing to reach the top.
 
-# Advanced topic AI: 
+## Advanced topic AI: 
 main implementation: AI created should be a deep learning model focused as a cooperative player for platforming, using a behavior tree to solve jumping with 2 or more players. 
```

Comment:
Need way more details about what you're going to be building here. What approaches to use, how to train, etc.